azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeIncomingCallEndEvent.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.models;
```

Comment:
Should we have a separate space under models for the events and related event types which will make the directory be like `ui.calling.models.events` or `ui.calling.models.pushevents`